### PR TITLE
Display picture on guess activity

### PR DESCRIPTION
**Pull request by rkuenzi-epfl**

### CODE REVIEW

File: app/src/main/java/com/github/wnder/picture/ExistingPicture.java [hunk 1]
```diff
@@ -31,7 +34,11 @@
 
     public ExistingPicture(String uniqueId){
         super(uniqueId);
-        addToUserGuessedPictures(GlobalUser.getUser().getName());
+        User user = GlobalUser.getUser();
+        if(user instanceof SignedInUser){
+
+            addToUserGuessedPictures(user.getName());
+        }
```
Comment (BoostedBoat) [inline app/src/main/java/com/github/wnder/picture/ExistingPicture.java:41]: Do we want to add to the guessed pictures as soon as a user "sees" a picture ? Or do we want to wait for him to press on the "guess picture" button ? (or maybe I didn't understand something)

**Review by BoostedBoat — CHANGES_REQUESTED** [review]
Seems globally nice to me, just a little conceptual question about when to add a picture to a user's guessed pictures

File: app/src/main/java/com/github/wnder/picture/ExistingPicture.java (hunk 1 above)
Comment (rkuenzi-epfl) [reply app/src/main/java/com/github/wnder/picture/ExistingPicture.java:41]: Good question! Then we also need to decide what to do with skipped and reported pictures

Comment (leonardvaney) [reply app/src/main/java/com/github/wnder/picture/ExistingPicture.java:41]: Skipped, reported or guessed pictures are all pictures that the user has already encounter, so they should be the one added to "guessedPictures".

Comment (rkuenzi-epfl) [reply app/src/main/java/com/github/wnder/picture/ExistingPicture.java:41]: Then I guess @BoostedBoat can approve the review?

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java [hunk 2]
```diff
@@ -2,13 +2,28 @@
...
+
+import java.util.concurrent.ExecutionException;
+
+public class GuessPreviewActivity extends AppCompatActivity{
+
+    public static final String EXTRA_GUESSLAT = "guessLat";
+    public static final String EXTRA_GUESSLNG = "guessLng";
+    public static final String EXTRA_CAMERALAT = "cameraLat";
+    public static final String EXTRA_CAMERALNG = "cameraLng";
+
```
Comment (leonardvaney) [inline app/src/main/java/com/github/wnder/GuessPreviewActivity.java:26]: Unused

**Review by leonardvaney — DISMISSED** [review]
There is some unused import on the files you work on, you should use Ctrl+Alt+O to remove them easily.

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java (hunk 2 above)
Comment (rkuenzi-epfl) [reply app/src/main/java/com/github/wnder/GuessPreviewActivity.java:26]: they ar used on line 59-62

**Review by JeremyDezalos — APPROVED** [review]

**Review by amattei777 — APPROVED** [review]
Approval of Leonard comment about unsused import (He asked me to)

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java (hunk 2 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/GuessPreviewActivity.java:23]: indeed

File: app/src/main/java/com/github/wnder/picture/ExistingPicture.java (hunk 1 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/picture/ExistingPicture.java:37]: I agree with @leonardvaney and @JeremyDezalos after discussing with him agrees too.

**Pull request merged**